pyproject.toml

```diff
@@ -4,18 +4,18 @@ version = "0.4.2"
 description = "An easy-to-use Python library for building recommendation systems"
 license = "Apache-2.0"
 authors = [
+    "Emiliy Feldman <feldlime@yandex.ru",
```

Comment:
Please, close the bracket here and in the next section :)

Reply:
fixed